addons/web/static/src/scss/form_view.scss

```diff
@@ -661,10 +662,6 @@
     margin-right: 4px;
 }
 
-// Light labels (not in bold) for settings forms
-label.o_light_label, .o_light_label label {
-    font-weight: normal;
-}
```

Comment:
Is this necessary to move ?

Reply:
@qsm-odoo , Yes, if we do not move in o_settings_container then it will be overridden by o_form_label. Thanks